build/media_source/com_joomlaupdate/joomla.asset.json

```
@@ -6,51 +6,50 @@
   "license": "GPL-2.0-or-later",
   "assets": [
     {
-      "name": "com_joomlaupdate.admin-update",
+      "name": "com_joomlaupdate.admin-update-es5",
       "type": "script",
-      "uri": "com_joomlaupdate/admin-update-default.min.js",
+      "uri": "com_joomlaupdate/admin-update-default-es5.min.js",
       "dependencies": [
         "core",
-        "jquery"
+        "bootstrap.modal"
       ],
       "attributes": {
-        "defer": true
+        "defer": true,
+        "nomodule": true
       }
     },
     {
-      "name": "com_joomlaupdate.default",
+      "name": "com_joomlaupdate.admin-update",
       "type": "script",
-      "uri": "com_joomlaupdate/default.min.js",
+      "uri": "com_joomlaupdate/admin-update-default.min.js",
       "dependencies": [
-        "core",
-        "jquery"
+        "com_joomlaupdate.admin-update-es5"
       ],
       "attributes": {
-        "defer": true
+        "type": "module"
       }
     },
     {
-      "name": "com_joomlaupdate.encryption",
+      "name": "com_joomlaupdate.default-es5",
       "type": "script",
-      "uri": "com_joomlaupdate/encryption.min.js",
+      "uri": "com_joomlaupdate/default-es5.min.js",
       "dependencies": [
-        "core"
+        "core",
```

```suggestion
        "core"
```

The comma here will throw, sorry my bad

---

You could work off your guilt with a test later 😉

---

Yup, do a final test to give us the all good and all systems are go for merging. Phew 😅

---

Sorry for the very late review...

---

modern times, it's called "just in time delivery" 😄